File: vs2003_cd01/Program Files/Microsoft Visual Studio .NET 2003/SDK/v1.1/Samples/Tutorials/clr_fundamentals/samples/typefundamentals/objectbasetype/cs/ObjectBaseType.cs
using System;

// This type is implicitly derived from System.Object
class SimpleType {
}

class App {
    public static void Main() {
        SimpleType st = new SimpleType();
        Console.WriteLine("Equals object: "  + st.Equals(new Object()));
        Console.WriteLine("GetHashCode: "    + st.GetHashCode());
        Console.WriteLine("ToString: "       + st.ToString());
        Console.WriteLine("GetType: "        + st.GetType());

	Console.WriteLine();
	Console.Write("Press Enter to close window.");
	Console.Read();
    }	
}

File: vs2003_cd01/Program Files/Microsoft Visual Studio .NET 2003/SDK/v1.1/Samples/Tutorials/clr_fundamentals/samples/typefundamentals/objectbasetype/vb/ObjectBaseType.vb
Imports System

Class SimpleType
End Class

Class App
    Public Shared Sub Main()
        Dim st As New SimpleType()
        Console.WriteLine("Equals object: " + st.Equals(New Object()).ToString())
        Console.WriteLine("GetHashCode: " + st.GetHashCode().ToString())
        Console.WriteLine("ToString: " + st.ToString())
        Console.WriteLine("GetType: " + st.GetType().ToString())
        
        Console.WriteLine()
	Console.Write("Press Enter to close window.")
        Console.Read()
    End Sub
End Class